src/patternfly/components/AboutModalBox/styles.scss

```diff
@@ -0,0 +1,267 @@
+
+    &-image {
+      height: var(--pf-c-about-modal-box__brand-image--Height);
+    }
+  }
+
+  // Header
+  &__header {
+    grid-area: header;
+    display: flex;
+    flex-direction: column;
+    padding-right: var(--pf-c-about-modal-box__header--PaddingRight);
+    padding-bottom: var(--pf-c-about-modal-box__header--PaddingBottom);
+    padding-left: var(--pf-c-about-modal-box__header--PaddingLeft);
+
+    &-strapline {
+      padding-bottom: var(--pf-c-about-modal-box__header-strapline--PaddingBottom);
+    }
+  }
+
+  // Content
+  &__content {
+    grid-area: content;
+    min-height: var(--pf-c-about-modal-box__content--MinHeight);
+    // Using margin instead of padding to prevent scrollbar extending to top of content block
+    margin-top: var(--pf-c-about-modal-box__content--MarginTop);
+    margin-right: var(--pf-c-about-modal-box__content--MarginRight);
+    margin-bottom: var(--pf-c-about-modal-box__content--MarginBottom);
+    margin-left: var(--pf-c-about-modal-box__content--MarginLeft);
+    overflow-x: hidden;
+    overflow-y: auto;
+    overscroll-behavior: contain;
+  }
+
+  // Logo
+  &__logo {
+    grid-area: logo;
+    padding-right: var(--pf-c-about-modal-box__logo--PaddingRight);
+    padding-bottom: var(--pf-c-about-modal-box__logo--PaddingBottom);
+    padding-left: var(--pf-c-about-modal-box__logo--PaddingLeft);
+
+    @media only screen and (min-width: $pf-global--breakpoint--lg) {
+      grid-area: 2 / 2;
+      align-self: end;
+      padding: 0;
+    }
+
+    &-image {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: var(--pf-c-about-modal-box__logo-image--Width);
+      height: var(--pf-c-about-modal-box__logo-image--Height);
+      padding: var(--pf-c-about-modal-box__logo-image--PaddingTop) var(--pf-c-about-modal-box__logo-image--PaddingRight) var(--pf-c-about-modal-box__logo-image--PaddingBottom) var(--pf-c-about-modal-box__logo-image--PaddingLeft);
+      background-color: var(--pf-c-about-modal-box__logo-image--BackgroundColor);
+    }
+  }
+
+  // Close
+  &__close {
+    grid-area: close;
+    z-index: var(--pf-c-about-modal-box__close--ZIndex);
+    display: flex;
+    align-items: flex-start;
+    justify-content: flex-end;
+    padding-top: var(--pf-c-about-modal-box__close--PaddingTop);
+    padding-right: var(--pf-c-about-modal-box__close--PaddingRight);
+    padding-bottom: var(--pf-c-about-modal-box__close--PaddingBottom);
+
+    @media only screen and (min-width: $pf-global--breakpoint--sm) {
+      grid-area: 1 / 2;
+      // center the close over the narrow hero image
+      justify-content: center;
+      padding-right: 0;
+    }
+
+    @media only screen and (min-width: $pf-global--breakpoint--lg) {
+      // align flex end the close over the narrow hero image
+      justify-content: flex-end;
+      padding-right: var(--pf-c-about-modal-box__close--PaddingRight);
+    }
+
+    // Close button
+    .pf-c-button.pf-m-action {
```

Comment:
Since this btn is unique in the whole system, I am ok with the contextual selector. Can we make a comment on the code please?